addons/block_code/ui/main_panel.gd

```diff
@@ -55,7 +55,7 @@ func _on_undo_redo_version_changed():
 		return
 
 	var block_script: BlockScriptSerialization = _current_block_code_node.block_script
-	_picker.block_script_selected(block_script)
+	_picker.block_script_selected(block_script, _current_block_code_node.get_parent() if _current_block_code_node else null)
```

Comment:
The line above is assuming that `_current_block_code_node` is not null (it is referencing `block_script` in it.

In any case, this would be a bit more readable without a ternary operator.